pubsub/README.rst

```diff
@@ -41,6 +39,96 @@ Here's the rough usage pattern for using ``SubscriberClient``:
         max_messages=10)
 
 
+There's also ``gcloud.aio.pubsub.subscribe`` helper function you can use to
+setup a pubsub processing pipeline. It is built with ``asyncio`` and thus only
+available in ``gcloud-aio-pubsub`` package. The usage is fairly simple:
+
+.. code-block:: python
+
+    from gcloud.aio.pubsub import SubscriberClient
+    from gcloud.aio.pubsub import subscribe
+    from gcloud.aio.pubsub.metrics_agent import MetricsAgent
+
+    subscriber_client = SubscriberClient()
+
+    async def handler(message):
+        return
+
+    await subscribe(
+        'projects/<my_project>/subscriptions/<my_subscription>',
+        handler,
+        subscriber_client,
+        num_producers=1,
+        max_messages_per_producer=100,
+        ack_window=0.3,
+        num_tasks_per_consumer=1,
+        metrics_client=MetricsAgent()
+    )
+
+While defaults are somewhat sensible, it is highly recommended to performance
+test your application and tweak function parameter to your specific needs.
+Here's a few hints:
```

Comment:
I love this list of hints and how you linked parameter value to subscriber behaviour so the user knows which knob to turn when they see those behaviours.